src/toil/provisioners/aws/awsProvisioner.py

```diff
@@ -618,24 +648,28 @@ def _getBlockDeviceMapping(cls, instanceType, rootVolSize=50):
 
     @classmethod
     def _getNodesInCluster(cls, ctx, clusterName, nodeType=None, preemptable=False, both=False):
-        for attempt in retry(predicate=AWSProvisioner._throttlePredicate):
+        for attempt in retry(predicate=awsRetryPredicate):
```

Comment:
The decorator only works for a single retry in the function? Would it be cleaner to break up these functions and use the decorator?

Reply:
I considered it, but it seemed sort of unnecessary since this is still pretty concise as is. I'm happy to change it if you want me to.